modules/ROOT/pages/email/email-gmail.adoc

```diff
@@ -6,23 +6,23 @@ endif::[]
 
 
 [[short_description]]
-The examples here provide guidance on setting up a Gmail connection using Email Connector over the available protocols.
+These examples provide guidance for setting up a Gmail connection using Email Connector over the available protocols.
 
 == Gmail Configurations
```

Comment:
Suggest omitting everything up to the edit suggestion in Lines 17 through 20.
I left the original edits just to show the thought path....